app/views/users/_image_preview.html.erb

```diff
@@ -0,0 +1,25 @@
+<% if peerage.nil? %>
+  <div class="col-md-4 image-preview-container" id="arms_preview_container">
+    <div id="arms_preview">
+        <%= image_tag user.arms.url(:large) %>
+    </div>
+  </div>
+  <div class="col-md-4 image-preview-container hidden" id="profile_pic_preview_container">
+
+    <div id="profile_preview">
+        
+    </div>
+    <button class="btn btn-default hidden" id="crop">Finish</button>
+  </div>
+<% else %>
+  <div class="col-md-4 image-preview-container hidden" id="arms_preview_container">
+    <div id="arms_preview">
+    </div>
+  </div>
+  <div class="col-md-4 image-preview-container" id="profile_pic_preview_container">
+    <div id="profile_preview">
+      <%= image_tag user.peer(peerage).profile_pic.url(:large)%> 
+    </div>
+    <button class="btn btn-default hidden" id="crop">Finish</button>
+  </div>
+<% end %>
```

Comment:
Consider either cleaning up the JavaScript so it does't require the hidden divs. If that's not feasible, I think you can still remove the conditionals here with something like:

```erb
  <div class="col-md-4 image-preview-container <%= arms_preview_hidden %>" id="arms_preview_container">
    <div id="arms_preview">
        <%= arms_preview_image_tag %>
    </div>
  </div>
  <div class="col-md-4 image-preview-container <%= profile_preview_hidden %>" id="profile_pic_preview_container">

    <div id="profile_preview">
        <%= profile_preview_image_tag %>
    </div>
    <button class="btn btn-default hidden" id="crop">Finish</button>
  </div>
```

where `arms_preview_hidden` and `profile_preview_hidden` are either view helpers or variables passed in to the partial are either `"hidden"` or `""` based on whether `peerage` is `nil`, and `arms_profile_image_tag` and `profile_preview_image_tag` are either the empty string or the correct image tag based on whether `peerage` is `nil`.

Up to you if you want to fix this now or just make a note to do something about it later.

Reply:
I will put this off